plugins/dev-scripts/config/webpack.config.js

```diff
@@ -97,21 +97,9 @@ module.exports = (env) => {
           use: [require.resolve('source-map-loader')],
           enforce: 'pre',
         },
-        isTypescript ? {
+        isTypescript && {
```

Comment:
Side question for my own curiosity: why is the test here `.tsx` instead of `.ts`?

Reply:
It's tsx**?**, so should match both I belive.